_includes/download_sidebar.html

```diff
@@ -5,7 +5,9 @@
       {% if page.version == 'stable' %}<b><a>XCSoar {{ site.xcsoar_stable_version }}</a></b>{% else %}<a href="/download/index.html">XCSoar {{ site.xcsoar_stable_version }}</a>{% endif %}<br/>
       The current stable version.
     </li>
-    {% if site.xcsoar_testing_version != site.xcsoar_stable_version %}
+    {% assign testing_version_str = site.xcsoar_testing_version | downcase %}
+    {% assign stable_version_trunc_str = site.xcsoar_stable_version | downcase | slice: 0,testing_version_str.size %}
+    {% if testing_version_str != stable_version_trunc_str %}
```

Comment:
I'm not sure if that is the right solution 🤔 

Can't we just set the `xcsoar_testing_version` variable to `7.0.0-beta.1` or whatever is the current beta release?

Reply:
these is none yet (only Max's preview releases for Android, distributed via Play Stor). If one would exist, it would be correct to show this page.

What exactly is wrong with this check?

Reply:
if `6.8.8` is the most current release then I'd recommend to just set `testing_version_str` to `6.8.8` too or set it to `null` or whatever the equivalent is in Ruby

Reply:
corrected